metricbeat/docker-compose.yml

```diff
@@ -1,12 +1,12 @@
-version: '2'
```

Comment:
@exekias It seems the docker/libcompose does not support composer files of the version 2.1. Not sure if the project is still actively maintained. I reverted to version 2 as it does not make a difference for us at the moment I think.

Reply:
I'm afraid that's the case, in the future this may be an issue, but for the way we use compose today it should be OK